src/Components/SingleReview.jsx

```diff
@@ -27,15 +27,13 @@ function SingleReview() {
     };
     setCurrentReview(updatedReview);
     voteOnSingleReview(review_id, increment).catch((err) => {
-      setCurrentReview(currentReview);
-      setErrMessage("Error occurred while voting.");
-      console.err(err);
+      setErrMessage("Error occurred while voting. Your vote has not been counted. Please refresh the page and try again.");
+      console.log(err);
```

Comment:
This doesn't really need to be here however, anyone checking the console will see that the user is offline :D